src/Control/Parallel.purs

```diff
@@ -16,6 +18,16 @@ import Control.Parallel.Class (class Parallel, parallel, sequential, ParCont(..)
 import Data.Foldable (class Foldable, traverse_, oneOfMap)
 import Data.Traversable (class Traversable, traverse)
 
+parApply
+  :: forall f m a b
+   . Parallel f m
+   => m (a -> b)
+   -> m a
+   -> m b
+parApply mf ma = apply (parallel mf) (parallel ma) # sequential
+
+infixl 4 apply as <!>
```

Comment:
I agree it's pretty arbitrary. Personally I feel that an operator would provide a lot of value, but I'm ready to just define that in my own projects as an alias, if we choose not to include this :)